src/cascade/input_data/configuration/form.py

```diff
@@ -53,7 +53,17 @@ def _full_form_validation(self, root):  # noqa: C901 too complex
             if mean is None and np.isinf(lower) or np.isinf(upper):
                 mean = max(lower, 0)
             std = self.std
-            nu = self.nu
+
+            if self.nu is None:
+                if self.density == "students" and not root.is_field_unset("students_dof"):
+                    nu = root.students_dof.priors
+                elif self.density == "log_students" and not root.is_field_unset("log_students_dof"):
+                    nu = root.log_students_dof.priors
+                else:
+                    nu = None
+            else:
+                nu = self.nu
+
             if self.eta is None:
```

Comment:
similarly, for eta.  eta (maybe) helps prevent getting a log value of -inf, by keeping the value a little bit to the right of zero.